src/tokens.scss

```diff
@@ -0,0 +1,3 @@
+:host {
+  --ds-auro-icon-color: var(--ds-color-icon-primary-default, $ds-color-icon-primary-default);
```

Comment:
Is this still needed? I see the same line in the `styleColor.scss` file

Reply:
It was needed here and should have been removed from `styleColor.scss` file. This is updated.